components/service/glean/build.gradle

```diff
@@ -16,6 +16,12 @@ apply plugin: 'kotlin-android'
  */
 String GLEAN_PING_SCHEMA_GIT_HASH = "8939bcb"
 String GLEAN_PING_SCHEMA_URL = "https://raw.githubusercontent.com/mozilla-services/mozilla-pipeline-schemas/$GLEAN_PING_SCHEMA_GIT_HASH/schemas/glean/baseline/baseline.1.schema.json"
+// NOTE: this is also defined in scripts/sdk_generator.gradle
+String MINICONDA_VERSION = "4.5.11"
+String GLEAN_MINICONDA_DIR = new File(
+    project.getGradle().gradleUserHomeDir,
```

Comment:
It feels weird to put stuff in the gradle homedir :-\ Maybe another good location could be "parent's project build dir"? I'm not sure if such a thing is even possible though. @pocmo , any thought on that? The idea is to have a place where to download the environment for glean python stuff, to avoid having every a-c component using glean to have its own copy of the environment.

Reply:
I actually think it's not a bad place given what else already goes in `~/.gradle`: gradle itself, gradle plugins, caches downloads of packages.  It feels like the natural place to me -- moreso than the build directory, which I generally would think includes build products, not build-time tools.

Remember also, that this will be used by applications (and potentially libraries) that exist outside of a-c as well.

Reply:
Mh, I see. I still think @pocmo (or maybe @ncalexan ) might want to review this for sanity.

Reply:
I totally agree.  I'm a total newbie with gradle.